src/main/java/org/vaadin/addons/tatu/TabSheet.java

```diff
@@ -227,6 +228,15 @@ public String getTab(int index) {
         }
     }
 
+    /**
+     * Get selected tab identifier.
+     * @return Unique tab identifier or <code>null</code> if no selection.
+     */
+    public String getSelected() {
```

Comment:
getSelectedTab ?

Reply:
There are now three properties / method pairs: selected (id) , selectedIndex (index) and selectedTab (component). Naming of them I don't have a strong preference, and just followed what was already there. 

Actually, I could see the String ID setter/getter is mostly "internal API" because the IDs are generated internally. More typically you either want to set/get the component or its index.

Reply:
I left it untouched according to original API. I think one option to consider for the property naming is:
- selectedId (now: selected)
- selectedIndex (current PR)
- selectedComponent (current PR)

Reply:
Current naming is: 
- String selected
- int selectedIndex
- Component selectedComponent